CHANGELOG.md

```diff
@@ -101,7 +101,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - cmake: check for chflags() function and enable FreeBSD File Flags support [PR #963]
 - plugin: added mariabackup python plugin, added systemtest for mariabackup and updated systemtest for percona-xtrabackup [PR #967]
 - webui: add inchanger column to volume tables [PR #998]
-
+- packages: Build also for Rocky_8 [PR #992]
```

Comment:
that should be something like "Build EL_8 packages instead of CentOS_8 for RHEL 8 derivatives"

Reply:
Agreed

Reply:
Ok updated the comment.